internal/c/libqb.cpp

```diff
@@ -29451,44 +29522,60 @@ int32 gfs_open_com_syntax(qbs *fstr, gfs_file_struct *f) {
     // complete omitted options with defaults
     if (f->com_baud_rate == -1)
         f->com_baud_rate = 300;
+
     if (f->com_parity == -1)
         f->com_parity = 1;
+
     if (f->com_data_bits_per_byte == -1)
         f->com_data_bits_per_byte = 7;
+
     if (f->com_stop_bits == -1) {
         x = 10;
         if (f->com_baud_rate <= 110) {
             x = 20;
             if (f->com_data_bits_per_byte == 5)
                 x = 15;
         }
+
         f->com_stop_bits = x;
     }
+
     if (f->com_bin_asc == -1)
         f->com_bin_asc = 0;
+
     if (f->com_asc_lf == -1)
         f->com_asc_lf = 0;
+
     if (f->com_asc_lf == 1) {
         if (f->com_bin_asc == 0)
             f->com_asc_lf = 0;
     }
+
     if (f->com_rs == -1)
         f->com_rs = 0;
+
     if (f->com_cd_x == -1)
         f->com_cd_x = 0;
+
     if (f->com_cs_x == -1)
         f->com_cs_x = 1000;
+
     if (f->com_ds_x == -1)
         f->com_ds_x = 1000;
+
     if (f->com_op_x == -1) {
         x = f->com_cd_x * 10;
         z = f->com_ds_x * 10;

```

Comment:
I think this is the only usage of `z` (hard to tell). If it is you might as well just declare it right here.

Reply:
Done.